### PR TITLE
feat: validate donor_id

### PR DESCRIPTION
**Pull request by ebezzi**

### CODE REVIEW

File: cellxgene_schema_cli/tests/fixtures/examples_validate.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
             True,
             "HANCESTRO:0575",
             "HsapDv:0000003",
+            "na",
```
Comment (ebezzi) [inline cellxgene_schema_cli/tests/fixtures/examples_validate.py:48]: Could replace it with a more meaningful value, if applicable.

Comment (pablo-gar) [inline cellxgene_schema_cli/tests/fixtures/examples_validate.py:48]:
```suggestion
            "donor_1",
```

File: cellxgene_schema_cli/tests/fixtures/examples_validate.py [hunk 2]
```diff
@@ -56,6 +57,7 @@
             False,
             "na",
             "MmusDv:0000003",
+            "na",
```
Comment (pablo-gar) [inline cellxgene_schema_cli/tests/fixtures/examples_validate.py:60]:
```suggestion
            "donor_2",
```

**Review by pablo-gar — APPROVED** [review]
small nits otherwise looks good to me

**Pull request merged**